cfme/cloud/instance/__init__.py

```diff
@@ -423,6 +423,65 @@ def power_control_from_cfme(self, *args, **kwargs):
         view.toolbar.power.item_select(kwargs.get('option'),
                                        handle_alert=not kwargs.get('cancel', False))
 
+    def set_ownership(self, user=None, group=None, click_cancel=False, click_reset=False):
+        """Set instance ownership
+
+        TODO: collapse this back to common.vm after both subclasses converted to widgetastic
+        Args:
+            user (str): username for ownership
+            group (str): groupname for ownership
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
```

Comment:
**REQUIRED**
@mshriver I understand you based this on `cfme.common.vm.BaseVM.set_ownership` which has `click_cancel` and `click_reset` params, but it's not correct. We don't prefix `cancel` and `reset` params with `click_` anywhere else, and since this is a good opportunity to get rid of it in `set_ownership` as well, let's do exactly that, please. :blush: